app/main.py

```diff
@@ -16,4 +16,22 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [], "immutable": []}
+
+
+def sort_element(variables_dict: dict, *elements) -> None:
+    for el in elements:
+        if (
+                isinstance(el, list)
+                or isinstance(el, dict)
+                or isinstance(el, set)
+        ):
```

Comment:
you can write it like this
`if (isinstance(el, (list, dict, set)):` or `if (isinstance(el, list | dict | set):`